File: NetNodePluginServer/NetNodeClient/Service/ClientNodeService.py
from EtherealC.Service.WebSocket.WebSocketService import WebSocketService

from EtherealS.Net.NetNode.Model.NetNode import NetNode


class ClientNodeService(WebSocketService):
    def __init__(self):
        super().__init__()
        self.name = "ClientNetNodeService"
        self.types.add(type=int, type_name="Int")
        self.types.add(type=str, type_name="String")
        self.types.add(type=bool, type_name="Bool")
        self.types.add(type=type(NetNode()), type_name="NetNode")

File: NetNodePluginServer/NetNodeClient/Service/ClientNodeService.cs
using EtherealC.Core.Model;
using NetNodePlugin.Model;
using NetNodePlugin.NetNodeClient.Request;

namespace NetNodePlugin.NetNodeClient.Service
{
    public class ClientNodeService:EtherealC.Service.WebSocket.WebSocketService
    {
        #region --字段--

        public ClientNodeService()
        {
            name = "ClientNetNodeService";
            types.Add<int>("Int");
            types.Add<long>("Long");
            types.Add<string>("String");
            types.Add<bool>("Bool");
            types.Add<NetNode>("NetNode");
        }


        #endregion

        #region --属性--

        #endregion

        #region --RPC方法--

        #endregion

        #region --普通方法--

        public override void Initialize()
        {

        }


        public override void UnInitialize()
        {

        }
        #endregion


    }
}
